it/uk/gov/hmrc/helpers/controllers/SubscriptionControllerSpec.scala

```diff
@@ -226,4 +226,73 @@ class SubscriptionControllerSpec extends IntegrationSpec with AuthHelpers  {
 
   }
 
+  "return 200" when {
+
+    "AWRS feature flag is true" in {
+
+      val jsResultString =
+        """
+          |{
+          |  "sapNumber": "1234567890",
+          |  "safeId": "XE0001234567890",
+          |  "agentReferenceNumber": "AARN1234567",
+          |  "regimeIdentifiers": [
+          |    {
+          |      "regimeName": "AWRS",
+          |      "regimeRefNumber": "XAAW00000123456"
+          |    },
+          |    {
+          |      "regimeRefNumber": "XAML00000123456"
+          |    }
+          |  ],
+          |  "nonUKIdentification": {
+          |    "idNumber": "123456",
+          |    "issuingInstitution": "France Institution",
+          |    "issuingCountryCode": "FR"
+          |  },
+          |  "isEditable": true,
+          |  "isAnAgent": false,
+          |  "isAnIndividual": false,
+          |  "organisation": {
+          |    "organisationName": "ACME Trading",
+          |    "isAGroup": false,
+          |    "organisationType": "Corporate body"
+          |  },
+          |  "addressDetails": {
+          |    "addressLine1": "100 SomeStreet",
+          |    "addressLine2": "Wokingham",
+          |    "addressLine3": "Surrey",
+          |    "addressLine4": "London",
+          |    "postalCode": "DH14EJ",
+          |    "countryCode": "GB"
+          |  },
+          |  "contactDetails": {
+          |    "regimeName": "AWRS",
+          |    "phoneNumber": "01332752856",
+          |    "mobileNumber": "07782565326",
+          |    "faxNumber": "01332754256",
+          |    "emailAddress": "stephen@manncorpone.co.uk"
+          |  }
+          |}
+        """.stripMargin
+
+      //TODO: needs refactoring when completed solution in place as it will need to be passed into new function
```

Comment:
RED - I think you've already done these TODOs so I think just the comments need deleting.